pkg/roachpb/batch.go

```diff
@@ -164,17 +164,17 @@ func (ba *BatchRequest) IsIntentWrite() bool {
 	return ba.hasFlag(isIntentWrite)
 }
 
-// IsUnsplittable returns true iff the BatchRequest an un-splittable request.
-func (ba *BatchRequest) IsUnsplittable() bool {
-	return ba.hasFlag(isUnsplittable)
-}
-
 // ConsultsTimestampCache returns whether the request must consult
```

Comment:
Why keep `BatchRequest.ConsultsTimestampCache()` but not `ConsultsTimestampCache(Request)`? I'd either keep both or remove both.

Reply:
done